src/main/webapp/app/view/modal-outlet.routes.ts

```diff
@@ -8,13 +8,22 @@ import { ProjectResponsibilityModalComponent } from 'app/view/project/admin/resp
 import { ProjectUserModalComponent } from 'app/view/project/admin/users/project-user-modal.component';
 
 import { ProjectLocationModalComponent } from 'app/view/project/admin/locations/project-location-modal.component';
+import { ProjectAdminUpdateModalComponent } from 'app/view/project/admin/project-admin-update-modal.component';
```

Comment:
bitte die Auflistung gemäss der Verwendung unten in Reihenfolge bringen.. und ohne Leerzeilen :)